temporalcli/commandsmd/commands.md

```diff
@@ -1512,29 +1512,33 @@ Workflow and Activity polling use separate Task Queues:
 ```
 temporal task-queue describe \
     --task-queue YourTaskQueue \
-    --task-queue-type "activity"`
+    --task-queue-type "activity"
 ```
 
 This command provides the following task queue statistics:
 - `ApproximateBacklogCount`: The approximate number of tasks backlogged in this
-  task queue.  May count expired tasks but eventually converges to the right
+  task queue. May count expired tasks but eventually converges to the right
   value.
-- `ApproximateBacklogAge`: How far "behind" the task queue is running. This is
-  the difference in age between the oldest and newest tasks in the backlog,
-  measured in seconds.
+- `ApproximateBacklogAge`: Approximate age of the oldest task in the backlog,
+  based on its creation time, measured in seconds.
 - `TasksAddRate`: Approximate rate at which tasks are being added to the task
   queue, measured in tasks per second, averaged over the last 30 seconds.
   Includes tasks dispatched immediately without going to the backlog
-  (sync-matched tasks), as well as tasks added to the backlog.
+  (sync-matched tasks), as well as tasks added to the backlog. (See note below.)
 - `TasksDispatchRate`: Approximate rate at which tasks are being dispatched from
   the task queue, measured in tasks per second, averaged over the last 30
   seconds.  Includes tasks dispatched immediately without going to the backlog
-  (sync-matched tasks), as well as tasks added to the backlog.
+  (sync-matched tasks), as well as tasks added to the backlog. (See note below.)
 - `Backlog Increase Rate`: Approximate rate at which the backlog size is
   increasing (if positive) or decreasing (if negative), measured in tasks per
   second, averaged over the last 30 seconds.  This is equivalent to:
   `TasksAddRate` - `TasksDispatchRate`.
 
+NOTE: The `TasksAddRate` and `TasksDispatchRate` metrics may differ from the
+actual rate of add/dispatch, because tasks may be dispatched eagerly to an
+available worker, or may apply only to specific workers (they are "sticky").
+Such tasks are not counted by these metrics.
```

Comment:
Thanks for adding this. It looks great. I'd just add once sentence at the end that clarifies: "Despite the inaccuracy of these two metrics, the derived metric of `BacklogIncreaseRate` is accurate for backlogs older than few seconds."